components/malloc_size_of/Cargo.toml

```diff
@@ -47,3 +47,4 @@ url = { version = "2.0", optional = true }
 webrender_api = { git = "https://github.com/servo/webrender", optional = true }
 xml5ever = { version = "0.16", optional = true }
 void = "1.0.2"
+uuid = {version = "0.8", features = ["v4", "serde"]}
```

Comment:
Is `"serde"` necessary?

Reply:
I guess I can remove serde here.

Reply:
Please make this optional and enable it as part of the servo feature.

Reply:
I have removed `serde` here and made the other changes too.